qemu/tests/cfg/numa_cpu.cfg

```diff
@@ -79,3 +82,50 @@
                   numa_cpu_dieid_cpu4 = 1
                   numa_cpu_coreid_cpu4 = 1
                   numa_cpu_threadid_cpu4 = 1
+      - with_preconfig:
+          qemu_preconfig = on
+          # Ensure all node have corresponding cpu in qemu cli
+          x86_64, i386:
+              numa_cpu_nodeid_cpu0 = 0
+              numa_cpu_socketid_cpu0 = 0
+              numa_cpu_dieid_cpu0 = 0
+              numa_cpu_coreid_cpu0 = 0
+              numa_cpu_threadid_cpu0 = 0
+
+              numa_cpu_nodeid_cpu1 = 1
+              numa_cpu_socketid_cpu1 = 0
+              numa_cpu_dieid_cpu1 = 1
+              numa_cpu_coreid_cpu1 = 0
+              numa_cpu_threadid_cpu1 = 0
+
+              numa_cpu_nodeid_cpu2 = 2
+              numa_cpu_socketid_cpu2 = 1
+              numa_cpu_dieid_cpu2 = 0
+              numa_cpu_coreid_cpu2 = 0
+              numa_cpu_threadid_cpu2 = 0
+
+              numa_cpu_nodeid_cpu3 = 3
+              numa_cpu_socketid_cpu3 = 1
+              numa_cpu_dieid_cpu3 = 1
+              numa_cpu_coreid_cpu3 = 0
+              numa_cpu_threadid_cpu3 = 0
+          ppc64le, ppc64:
+              guest_numa_nodes = "node0 node1"
+              mem_devs = "mem0 mem1"
+              size_mem_mem0 = 2048M
+              size_mem_mem1 = 3072M
+              numa_memdev_node0 = mem-mem0
+              numa_memdev_node1 = mem-mem1
+              numa_nodeid_node0 = 0
+              numa_nodeid_node1 = 1
+              del size_mem_mem2
+              del size_mem_mem3
+              del numa_memdev_node2
+              del numa_memdev_node3
+              del numa_nodeid_node2
+              del numa_nodeid_node3
+              guest_numa_cpus = "cpu0 cpu1"
+              numa_cpu_nodeid_cpu0 = 0
+              numa_cpu_coreid_cpu0 = 0
+              numa_cpu_nodeid_cpu1 = 1
+              numa_cpu_coreid_cpu1 = 2
```

Comment:
Hi Yana,
Could you please add more numa topology as following, let us synchronize with the test plan . Thanks.
-m 5120
-object memory-backend-ram,size=512M,id=mem-mem0
-object memory-backend-ram,size=1024M,id=mem-mem1
-object memory-backend-ram,size=1536M,id=mem-mem2
-object memory-backend-ram,size=2048M,id=mem-mem3

-smp 8,maxcpus=8,cores=2,threads=1,sockets=4
-numa node,memdev=mem-mem0,nodeid=0
-numa node,memdev=mem-mem1,nodeid=1
-numa node,memdev=mem-mem2,nodeid=2
-numa node,memdev=mem-mem3,nodeid=3
-numa cpu,node-id=3,core-id=0
-numa cpu,node-id=2,core-id=1
-numa cpu,node-id=1,core-id=3
-numa cpu,node-id=0,core-id=4